doc/SpectatorView.Setup.md

```diff
@@ -125,14 +125,18 @@ If you are building Azure Spatial Anchors on iOS, you will need to take some add
 1. Make sure your Unity project contains the asset caches that were created in the 'Before building' steps.
 2. Open the project scene that you intend to use with SpectatorView.
 3. Add the `SpectatorView` prefab to the scene.
-4. Setup your scene to synchronize content. You can either have all content synchronized by checking 'Automatically Broadcast All Game Objects' in BroadcasterSettings located in your SpectatorViewSettings prefab. Or, you can manually add GameObjectHierarchyBroadcaster components to all parent game objects in the scene that you want synchronized.
+4. Add the `HoloLensTrackingObserver` MonoBehaviour to your HoloLens Camera.

```

Comment:
Why does this need to be added to the camera? Can we not just have this included in the SpectatorView prefab?

Reply:
Updating this